improver_tests/acceptance/test_phase_change_level.py

```diff
@@ -41,54 +41,30 @@
 run_cli = acc.run_cli(CLI)
 
 
-def test_snow_sleet(tmp_path):
-    """Test snow/sleet level calculation"""
+@pytest.mark.parametrize("phase_type,horiz_interp",
+                         [("snow-sleet", "True"),
+                          ("sleet-rain", "True"),
+                          ("sleet-rain-unfilled", "False")])
+def test_phase_change(tmp_path, phase_type, horiz_interp):
+    """Testing:
+        sleet/rain level
+        snow/sleet level
+        sleet/rain level leaving below orography points	unfilled.
```

Comment:
```suggestion
        sleet/rain level leaving below orography points unfilled.
```
rouge tab

Reply:
Sounds like a good idea, I'll sort it now